unit/analyses/variable-sensitivity/value_set/abstract_value.cpp

```diff
@@ -359,3 +361,117 @@ TEST_CASE(
 
   REQUIRE(value_set.to_constant() == value);
 }
+
+static abstract_environmentt get_value_set_abstract_environment()
+{
```

Comment:
One thing I really like about variable sensitivity that at least in principle most things here are much easier to test in isolation than other parts of the codebase.